src/views/Home.vue

```diff
@@ -1,86 +1,20 @@
 <template>
-    <div class="layout">
-        <Layout :style="{ minHeight: '100vh' }">
-            <Sider>
-                <Menu active-name="1-2" theme="dark" width="auto">
-                    <MenuItem name="1-1">
-                        <Icon type="ios-navigate"></Icon>
-                        <span>{{ $t('views.home.option1') }}</span>
-                    </MenuItem>
-                    <MenuItem name="1-2">
-                        <Icon type="search"></Icon>
-                        <span>{{ $t('views.home.option2') }}</span>
-                    </MenuItem>
-                    <MenuItem name="1-3">
-                        <Icon type="settings"></Icon>
-                        <span>{{ $t('views.home.option3') }}</span>
-                    </MenuItem>
-                </Menu>
-            </Sider>
-            <Layout>
-                <Header
-                    :style="{
-                        background: '#fff',
-                        boxShadow: '0 2px 3px 2px rgba(0,0,0,.1)',
-                    }"
-                    data-cy="top-navbar"
-                >
-                    <Button @click="changeLanguage()" type="primary">
-                        {{ $t('common.buttonTranslations') }}
-                    </Button>
-                </Header>
-                <Content :style="{ padding: '0 16px 16px' }">
-                    <Breadcrumb :style="{ margin: '16px 0' }">
-                        <BreadcrumbItem>
-                            {{ $t('views.home.home') }}
-                        </BreadcrumbItem>
-                        <BreadcrumbItem>
-                            {{ $t('views.home.components') }}
-                        </BreadcrumbItem>
-                        <BreadcrumbItem>
-                            {{ $t('views.home.layout') }}
-                        </BreadcrumbItem>
-                    </Breadcrumb>
-                    <Card>
-                        <div style="height: 600px">
-                            {{ $t('views.home.content') }}
-                        </div>
-                    </Card>
-                </Content>
-            </Layout>
-        </Layout>
+    <div class="home">
+        <Card>
+            <div style="height: 1200px">
```

Comment:
@Andresws12 And here the same. Is necesary apply in the element? or you can apply with css?